jenkins/map.jinja

```diff
@@ -5,6 +5,14 @@
         'group': 'jenkins',
         'home': '/var/lib/jenkins',
         'port': '80',
-        'server_name': None
+        'server_name': None,
```

Comment:
So, alternately, Jenkins offers jenkins-cli.jar that you can just download from your instance instead of guessing the path-- maybe file managed with no hash check? I'll look into this.